tests/test1.spec.ts

```diff
@@ -0,0 +1,44 @@
+import {expect, test } from '@playwright/test';
+
+
+test('Login to the site', async ({ page }) => {
+    await page.goto('/auth/login');
+
+    await page.getByPlaceholder('Your email').fill('customer@practicesoftwaretesting.com');
+    await page.getByPlaceholder('Your password').fill('welcome01');
+    await page.locator('[data-test="login-submit"]').click();
+    await page.waitForURL('/account');
+    await expect(page.locator('[data-test="page-title"]')).toContainText('My account');
+    await expect(page.locator('[data-test="nav-menu"]')).toHaveText(' Jane Doe ');
+
+});
+
+test('User can view products details', async ({ page }) => {
+    await page.goto('/');
```

Comment:
you can add this to beforeEach function to avoid copy-paste https://playwright.dev/docs/api/class-test#test-before-each

Reply:
Ooops, you can ignore this comment. Or split tests into several files and use beforeEach with those test starting with home page